lib/Db/TotpSecretMapper.php

```diff
@@ -56,4 +56,15 @@ public function getSecret(IUser $user) {
         return TotpSecret::fromRow($row);
     }
 
+	/**
+	 * @param boolean $status
+	 */
+    public function setAllSecretsVerificationStatus($status) {
+		/* @var $qb IQueryBuilder */
+		$qb = $this->db->getQueryBuilder();
+		$qb->update('twofactor_totp_secrets')
+			->set('verified', $qb->expr()->literal($status, IQueryBuilder::PARAM_INT))
```

Comment:
a boolean gets converted to int, fortunately PHP will convert false to 0 and true to 1 here :tophat:

Reply:
It is a fix for the failing test in PostgreSQL. The cause is same with owncloud/core#484. I tried PARAM_BOOL  and also this suggested solution https://stackoverflow.com/questions/12539460/postgresql-error-when-bitvalue-is-false?answertab=oldest#tab-top in here, but it is still failing. I will find an another way.